boa_engine/src/builtins/set/mod.rs

```diff
@@ -164,7 +164,8 @@ impl Set {
         Ok(set.into())
     }
 
-    /// Utility for constructing `Set` Objects.
+    /// Utility for constructing `Set` objects.
+    #[allow(clippy::unnecessary_wraps)]
```

Comment:
What is the reason for allowing this lint?
I think it would be better to return only a `JsObject` as the lint recommends

Reply:
It's good to maintain consistency when it doesn't affect the usability of our program. However, this change impacts the usability, because now you need to handle an error on a function that cannot error.

Reply:
Alright, I understand.